template/src/components/ui/Icon.tsx

```diff
@@ -0,0 +1,130 @@
+import { styled } from "@mui/material";
+import * as React from "react";
+
+type IIconNames = "close";
+
```

Comment:
Please add a description on how to use the icons and how to add new one:
- Copy SVG content
- Replace `<svg>` with a `<g>` element? Is this required?
- Camel case the attributes inside the SVG
- Any missing steps?

Reply:
[add icon usage description to readme](https://github.com/allaboutapps/cra-template-aaa-typescript/pull/23/commits/3a6550b0d591f0963bdd8fb8e2b43734ac903c3d)